packages/desktop-client/src/components/settings/Themes.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { themeNames, useTheme } from '../../style';
+import { View, Select, Text } from '../common';
+
+import { Setting } from './UI';
+
+export default function ThemeSettings({ saveGlobalPrefs }) {
+  let theme = useTheme();
+  return (
+    <Setting
+      primaryAction={
+        <View>
+          <Select
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏missing background color

<img width="562" alt="Screenshot 2023-07-20 at 08 25 45" src="https://github.com/actualbudget/actual/assets/886567/41d8c303-c7f3-48eb-9b83-fd02f253a92a">